### PR TITLE
CVRVU-17: birthdate updates

### PR DESCRIPTION
**Pull request by jdolkowski**
https://sd-vxnaid.atlassian.net/browse/VRVU-17

While I was working on a different feature. I saw that estimated date is not displayed in participant summary. I also made birth date mandatory.

### CODE REVIEW

File: app/src/main/java/com/jnj/vaccinetracker/register/dialogs/BirthDatePickerDialog.kt [hunk 1]
```diff
@@ -29,16 +28,17 @@ class BirthDatePickerDialog(
...
-            val c = Calendar.getInstance()
             val year = c.get(Calendar.YEAR)
             val month = c.get(Calendar.MONTH)
             val day = c.get(Calendar.DAY_OF_MONTH)
             datePicker.init(year, month, day, null)
         }
 
         dialog.findViewById<Button>(R.id.btn_ok).setOnClickListener {
-            selectedDate = DateTime(datePicker.year, datePicker.month, datePicker.dayOfMonth)
+            selectedDate = DateTime(datePicker.year, datePicker.month + 1, datePicker.dayOfMonth)
```
Comment (jdolkowski) [inline app/src/main/java/com/jnj/vaccinetracker/register/dialogs/BirthDatePickerDialog.kt:41]: datePicker.month was always 1 less than actual month

File: app/src/main/java/com/jnj/vaccinetracker/participantflow/screens/ParticipantFlowMatchingViewModel.kt [hunk 2]
```diff
@@ -212,6 +212,7 @@ class ParticipantFlowMatchingViewModel @Inject constructor(
                         participantId = participant.participantId,
                         irisMatchingScore = participant.matchingScore,
                         birthDateText = participant.birthDate.toDateTime().format(DateFormat.FORMAT_DATE),
+                        isBirthDateEstimated=participant.isBirthDateEstimated,
```
Comment (sherrif10) [inline app/src/main/java/com/jnj/vaccinetracker/participantflow/screens/ParticipantFlowMatchingViewModel.kt:215]: Are all these changes part of this commit.

Comment (jdolkowski) [reply app/src/main/java/com/jnj/vaccinetracker/participantflow/screens/ParticipantFlowMatchingViewModel.kt:215]: yes

**Review by sherrif10 — APPROVED** [review]
Thanks @jdolkowski cc @pwargulak

**Pull request merged**